src/plugins/intel_gpu/src/kernel_selector/cl_kernels/resample_opt.cl

```diff
@@ -251,9 +261,15 @@ KERNEL (resample_opt)(__global INPUT0_TYPE* input,
         const int ix = floor((x + out_x) * SCALES[4]);
         const int iy = floor(y * SCALES[3]);
 
+#if OUTPUT_DIMS == 5
+        const int iz = floor(z * SCALES[2]);
+        in_vec_t res = READ_FUNC(input, INPUT0_GET_INDEX(b, feature_block, iz, iy, ix));
+#else
         in_vec_t res = READ_FUNC(input, INPUT0_GET_INDEX(b, feature_block, iy, ix));
+#endif
 #elif defined(SAMPLE_TYPE_INTERP)
-    unroll_for (uint out_x = 0; out_x < OUTPUT_X_BLOCK_SIZE; out_x++) {
+    unroll_for (uint out_x = 0; out_x#if defined(SAMPLE_TYPE_CAFFE_INTERP)
```

Comment:
What is this line?

Reply:
something happened by accident, strange is that tests didn't fail. Fixed